Terminal.Gui/Views/TimeField.cs

```diff
@@ -96,7 +95,7 @@ void TimeField_Changed (object sender, ustring e)
 		/// </summary>
 		/// <remarks>
 		/// </remarks>
-		public DateTime Time {
+		public TimeSpan Time {
```

Comment:
What's the logic for using `TimeSpan1` vs. `DateTime` here?

Reply:
Because as suggested by @grahamehorner if control is only over `Time`, it doesn't make sense to have the `Date`.